docs/site/Model.md

```diff
@@ -190,9 +188,198 @@ class Product extends Entity {
 }
 ```
 
-Additionally, the model decorator is able to build the properties object through
-the information passed in or inferred by the property decorators, so the
-properties key value pair can also be omitted.
+However, the model decorator in lb4 is not exactly the same as what it is in
+lb3. For example, in
+[lb3](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html) we can
+pass in a model definition syntax in model decorator, such as properties,
+options, relation etc. But not all these entries are available in lb4 model
+decorator:
+
+In lb4, we only support `properties` and `setting` in the ModelDefinitionSyntax
+for now. Those `top-level properties` in lb3 now are passed in `settings`.
+[Options](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#options)
+in lb3 doesn't have the mapping feature in lb4 yet.
+
+As for entries in `settings`, lb4 supports these built-in entries for now:
+
+#### Available Entries of Settings
+
+  <table>
+  <thead>
+  <tr>
+    <th width="160">Property</th>
+    <th width="100">Type</th>
+    <th>Default</th>
+    <th>Description</th>
+  </tr>
+  </thead>
+
+  <tbody>
+  <tr>
+  <td><code>base</code></td>
+  <td>String</td>
+  <td>None</td>
+  <td>
+    Name of another model that this model extends. The model will "inherit" properties and methods of the base model.
+  </td>
+  </tr>
+
+  <tr>
+  <td><code>description</code></td>
+  <td>String or Array</td>
+  <td>None</td>
+  <td>
+    Optional description of the model.<br/><br/>
+    You can split long descriptions into arrays of strings (lines) to keep line lengths manageable; for example:
+  <pre style="font-size: 75%;">[ "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+
+"sed do eiusmod tempor incididunt ut labore et dolore",<br> "magna aliqua." ]
+
+</pre>
+    </td></tr>
+
+  <tr>
+  <td><code>excludeBaseProperties</code></td>
+  <td>Array</td>
+  <td><code>['id', 'password']</code></td>
+  <td>
+  Excludes the given list of properties from the base model from being visible. Use this instead of the approach documented under 'Exclude properties from base model' section below.
+  </td>
+  </tr>
+
+  <tr>
+  <td><code>forceId</code></td>
+  <td>Boolean</td>
+  <td><code>true</code></td>
+  <td>
+    If true, prevents clients from setting the auto-generated ID value manually.
+  </td>
+  </tr>
+
+  <tr>
+  <td><code>http</code></td>
```

Comment:
`http` is not supported in LB4. These settings are used by strong-remoting (REST API layer) in LB3, LB4 has a very different approach for REST APIs.

Reply:
I am not sure if `http` here is the same as REST APIs'. This value is handled [here](https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/model-builder.js#L132).

Reply:
@agnes512 In LB4 the http configurations are inferred from controller members and the rest server.

Reply:
AFAICT, juggler is reading only `pathName` from the `http` field, it's used to set `http` property on the model class elsewhere: https://github.com/strongloop/loopback-datasource-juggler/blob/6b83b6f4d1a4941917ffbaf8cebf476e40a53f83/lib/model-builder.js#L226-L230

The REST API layer in LB4 does not use the `http` field from juggler model classes, it does not even have access to them!

Let's move `http` to "unsupported" list.